src/app/features/users-page/users-page.component.html

```diff
@@ -0,0 +1,14 @@
+<a routerLink="/" class="back-link">Back To Home</a>
+<h1>List of all users:</h1>
+<table style="width: 1000px; border: 1px solid black">
+    <tr>
+        <th>NAME</th>
+    </tr>
+    <tbody *ngIf="users$ | async as users">
+        <tr *ngFor="let user of users">
+            <th>
+                <a [routerLink]="['/profile/' + user.firstName]">{{ user.firstName }}</a>
```

Comment:
if you have links like this with user id you need to emulate a method which get user by id, so rename `getProfile ()` to `getUser (id: string)` and just use id as `seed`